lib/puppet/type/rds_instance.rb

```diff
@@ -0,0 +1,184 @@
+Puppet::Type.newtype(:rds_instance) do
+  @doc = 'type representing an RDS instance'
```

Comment:
We're moving all of the descriptions to leading caps and a full stop at the end to match the default description for ensure. If you could switch them over. Very picky.